### DISCUSSION TITLE
Where is the Horizontal setting for Bar Charts?

### DISCUSSION THREAD

**Discussion by chrisshaw** [Budibase/budibase#2569] [Help] [answered]
The [docs](https://docs.budibase.com/design/components/charts#bar-chart) describe the Bar Chart component with the ability to change it from column to bar:

![image](https://user-images.githubusercontent.com/2966918/132586572-755ce94f-2759-46f0-b6ae-5040e03a854c.png)

But I can't find this in the actual settings:

![image](https://user-images.githubusercontent.com/2966918/132586773-21f41fcc-2916-48ba-a9c7-47a951746ebf.png)

**Comment by joebudi** [comment, marked as answer]
Hey Chris. Apologies, this is an error on our part. There should be no mention of a horizontal bar chart in the docs. We used to have this feature, but we removed it. I will update the docs now.

**Reply by chrisshaw** [reply]
Thanks for the update @joebudi . Maybe I should switch this to a new component request? What prompted removing it?

I have a use case where multiple column charts don't scale with the number of categories. In the report I'm trying to port to budibase I have them as bar charts.

I used GRID.is for this. It looks like appsmith's implementation (based on fusion charts) has this solved.

**Reply by joebudi** [reply]
Hey @chrisshaw . Yes, I'll mark this as a feature request. We use to use Fusion Charts - including the horizontal bar chart - but moved away from it due to performance issues and ... design. When we moved, we forgot to re-add the horizontal chart.

To be honest, it's not a big implementation. When do you need it for?

**Reply by chrisshaw** [reply]
Thanks @joebudi We have a high priority client review next week, and a critical one 10/4.

**Reply by joebudi** [reply]
We've readded the horizontal bar chart and it will go live with the next release at the end of the month.

**Reply by chrisshaw** [reply]
🚀 Sounds great! Thanks.

**Reply by joebudi** [reply]
https://github.com/Budibase/budibase/pull/2584
